pdns/dnsdist-lua.cc

```diff
@@ -91,13 +91,13 @@ class LuaResponseAction : public DNSResponseAction
   func_t d_func;
 };
 
-typedef boost::variant<string,vector<pair<int, string>>, std::shared_ptr<DNSRule> > luadnsrule_t;
+typedef boost::variant<string, vector<pair<int, string>>, std::shared_ptr<DNSRule>, DNSName, vector<pair<int, DNSName>> > luadnsrule_t;
```

Comment:
Not your doing but this line should go, it's redundant with the one in `dnsdist-lua.hh`